index.html

```diff
@@ -508,6 +509,7 @@ <h4>The MediaStreamTrackAudioStats interface</h4>
   readonly attribute DOMHighResTimeStamp deliveredFramesDuration;
   readonly attribute unsigned long long totalFrames;
   readonly attribute DOMHighResTimeStamp totalFramesDuration;
+  readonly attribute DOMHighResTimeStamp? latency;
```

Comment:
Since we're planning on adding more latency metrics in https://github.com/w3c/mediacapture-extensions/issues/128, do you prefer I rename this to `currentLatency`? @padenot @jan-ivar